### PR TITLE
CXP-1287: JSON Meta Schema V0.0.3

### PR DESCRIPTION
**Pull request by masacc**
**Description (for Contributor and Core Developer)**

New version of JSON Meta Schema, allowing number and boolean attribute types.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Comment by tseho** [comment]
What about https://github.com/akeneo/pim-community-dev/pull/18820 ?

**Comment by masacc** [comment]
> What about #18820 ?

Yes I know but I have to start this one from OCT-176 and as master is frozen, I can't wait for merge. We'll have some rework to do...

**Review by eoliveauakeneo — APPROVED** [review]

**Review by hogren — APPROVED** [review]

File: components/catalogs/back/tests/Integration/Infrastructure/Validation/ProductSchema/invalid/0.0.1-with-missing-target-type.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://api.akeneo.com/mapping/product/0.0.1/schema",
-  "description": "The required properties (type) are missing at #/properties/price",
+  "description": "The required properties (type) are missing at #/properties/name",
   "properties": {
-    "price": {}
```
Comment (tseho) [inline components/catalogs/back/tests/Integration/Infrastructure/Validation/ProductSchema/invalid/0.0.1-with-missing-target-type.json:5]: why ?

Comment (masacc) [reply components/catalogs/back/tests/Integration/Infrastructure/Validation/ProductSchema/invalid/0.0.1-with-missing-target-type.json:5]: Not very important but I did that to anticipate the future introduction of price attribute and keep consistency across future schemas. I can rollback if you disagree.

Comment (tseho) [reply components/catalogs/back/tests/Integration/Infrastructure/Validation/ProductSchema/invalid/0.0.1-with-missing-target-type.json:5]: It's probably dumb, but I'm not comfortable with the fact that we change tests of past schemas.

What's the issue with using price in those tests ?
Are you allowed to use schema 0.0.1, ask for a price as string, and receive `"39.99€"` ?

Comment (masacc) [reply components/catalogs/back/tests/Integration/Infrastructure/Validation/ProductSchema/invalid/0.0.1-with-missing-target-type.json:5]: ok, I rollback this changes

**Pull request merged**